apps/server/src/utils/coerceType.ts

```diff
@@ -90,7 +90,12 @@ export function coerceColour(value: unknown): string {
     if (!isColourHex(lowerCaseValue)) {
       throw new Error('Invalid hex colour received');
     }
-  } else if (!(lowerCaseValue in cssColours)) {
+    return lowerCaseValue;
+  }
+  if (lowerCaseValue === 'none') {
+    return ''; // None colour the same as the UI 'Ø' button
+  }
```

Comment:
why use a 'none' string instead of accepting a falsy value like `null` or `''`

Reply:
We could do that 
was thinking this was good be course it would reject accidental empty values
Do you think it's better to leave it loose?

Reply:
You are correct that you should be careful with undefined, since that would mean that we could accidentally override the value if property is not in the patch

But if we can verify that the property exists, I think it is best to accept `''` since that is what our data looks like. I believe that translating `'none'` to meaning `''` becomes a bit of a secret handshake and adds more complexity